Neos.Fusion/Migrations/Code/Version20161201202543.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace Neos\Flow\Core\Migrations;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+/**
+ * Moves fusion files from old path ``Resources/Private/TypoScript/`` to the new path
+ * ``Resources/Private/Fusion``
+ */
+class Version20161201202543 extends AbstractMigration
+{
+
+    public function getIdentifier()
+    {
+        return 'Neos.Fusion-20161201202543';
+    }
+
+    /**
+     * @return void
+     */
+    public function up()
+    {
+        $this->moveFile('Resources/Private/TypoScript/*', 'Resources/Private/Fusion');
+        $this->searchAndReplace('Resources/Private/TypoScript/', 'Resources/Private/Fusion/', ['ts2', 'fusion']);
```

Comment:
But this might be an issue: 
```
$this->searchAndReplace('Resources/Private/TypoScript/', 'Resources/Private/Fusion/', ['ts2', 'fusion']);
```
 does not apply on somethink like this:
```
include: resource://Neos.Fusion/Private/TypoScript/Root.fusion
```